source/includes/azure/_public_ip_addresses.md

```diff
@@ -96,5 +99,47 @@ Attributes | &nbsp;
 `sku`  <br /> *string* | The sku of the public IP. Possible values: `BASIC`, `STANDARD`.
 `domainName` <br/>*string* | The subdomain part of the fqdn. This is only present if one is defined.
 `fqn` <br/>*string* | The fqdn which points to the public IP.
+`idleTimeout` <br/>*integer* | The number of minutes for the idleTimeout. I can be between 4 and 30 minutes.
 `allocationMethod` <br/>*string* | Allocation method of the public IP address. Possible values: `DYNAMIC`, `STATIC`.
 `ipVersion` <br/>*string* |  IP version of the public IP address. Possible values: `IPV4`, `IPV6`.
+
+
+<!-------------------- CREATE A PUBLIC IP -------------------->
+
+#### Create a public ip address
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body"
+   "https://cloudmc_endpoint/v1/services/azure/example/publicipaddresses"
+
+# Request Example:
+```
+
+```json
+{
+	"name":"samplePublicIP",
+	"region" : "canadacentral",
+	"sku": "BASIC",
+	"allocationMethod" : "DYNAMIC",
+	"idleTimeout" : 30,
+	"domainName" : "samplePublicIP"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/publicipaddresses/:id</code>
```

Comment:
I don't think you put the :id at the end of the URL for POST

Reply:
e280c51a259109e604c2b41b10bbf75ea66b31e7